native/packages/react-native-bpk-component-navigation-bar/src/TitleView-test.common.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import TitleView from './TitleView';
+
+const commonTests = () => {
+  describe('TitleView', () => {
+    it('should render correctly with only a title', () => {
+      const tree = renderer
+        .create(<TitleView title="Backpack" tintColor="rgb(255, 255, 255)" />)
+        .toJSON();
+
+      expect(tree).toMatchSnapshot();
+    });
+
+    it('should render correctly with leading icon', () => {
```

Comment:
@shaundon / @matteo-hertel  looks like trailing / leading is already being tested here?

Reply:
damn, that's true, I'll remove the one I added 😞